src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -134,10 +136,7 @@ public function execute(WorkflowInstance $workflow)
             ]);
         }
 
-
-        $view = SSViewer::fromString($this->EmailTemplate);
-        $this->extend('updateView', $view);
```

Comment:
Not sure what anyone would have used that extension point for, but it wouldn't be useful now we're not using `SSViewer` at all anyway.